src/components/layout/DefaultLayout.tsx

```diff
@@ -110,16 +109,21 @@ export const DefaultLayoutHeader: React.FC<{
   const { formatMessage } = useIntl()
   const { currentMemberId, permissions, currentUserRole } = useAuth()
   const { enabledModules, id: appId, endedAt: appEndedAt, options: appOptions } = useApp()
+  const { appPlan, appPlanLoading } = useAppPlan()
   const { currentLocale, setCurrentLocale } = useContext(LocaleContext)
   const { totalVideoDuration, totalWatchedSeconds } = useAppUsage([moment().startOf('M'), moment().endOf('M')])
   const [updateAppOptions] = useMutation<hasura.UPDATE_APP_OPTIONS, hasura.UPDATE_APP_OPTIONSVariables>(
     UPDATE_APP_OPTIONS,
   )
 
   const isSiteExpiringSoon = dayjs(appEndedAt).diff(dayjs(), 'day') <= 20
-  const isVideoDurationExceedsUsage = Math.round(totalVideoDuration / 60) > appOptions.video_duration
-  const isWatchedSecondsExceedsUsage = totalWatchedSeconds > appOptions.video_duration
-  const closeSiteAt = appOptions.close_site_at
+  const isVideoDurationExceedsUsage =
+    (appPlan.options.maxVideoDurationUnit === 'minute' ? Math.round(totalVideoDuration / 60) : totalVideoDuration) >
+    appPlan.options.maxVideoDuration
+  const isWatchedSecondsExceedsUsage =
+    (appPlan.options.maxVideoDurationUnit === 'minute' ? Math.round(totalWatchedSeconds / 60) : totalWatchedSeconds) >
+    appPlan.options.maxVideoWatch
```

Comment:
It would result in an error when accessing `appPlan.options.{key}` because the options column of the `enterprise version` is null.

Reply:
There's a default value, so I think it shouldn't cause an error.